src/work-allocation-2/components/task-list-filter/task-list-filter.component.ts

```diff
@@ -2,10 +2,13 @@ import { Component, EventEmitter, OnDestroy, OnInit, Output } from '@angular/cor
 import { ActivatedRoute } from '@angular/router';
 import { FilterService } from '@hmcts/rpx-xui-common-lib';
 import { FilterConfig, FilterFieldConfig, FilterSetting } from '@hmcts/rpx-xui-common-lib/lib/models/filter.model';
+import { select, Store } from '@ngrx/store';
 import { Subscription } from 'rxjs';
 import { filter } from 'rxjs/operators';
+import { AppUtils } from '../../../app/app-utils';
```

Comment:
Very minor comment - Could this import be put with others below rather than with the ones above?